### DISCUSSION TITLE
Reorder List

### DISCUSSION THREAD

**Discussion by maxoakland** [gingerbeardman/stapler#31] [Q&A]
**Is your feature request related to a problem? Please describe.**
Right now the list can't be reordered. It appears to order items alphabetically 

**Describe the solution you'd like**
I'd like to be able to move items up and down so I can choose the order they open

**Additional context**
This would mean I could choose the order the windows appear in and pick the first thing I want to work on to open last so it's right up on my screen and ready to use

**Comment by gingerbeardman** [comment]
I had this at one point but the only way I could get consistent behaviour was to force alphabetical sorting.

Prior to that I had
- A-Z
- Z-A
- Manual (order they were added to the document)

But the process of saving to disk changed the order, so the reopened document had a different order. So given that there was a lot more work involved I decided to force A-Z.

Currently drag is used to select multiple items in the list, so adding ordering would require even more work.

If you could let me know more about your use case? Can you give an example of when opening order maters?

I'll consider it.
